src/main/resources/com/google/api/codegen/py/README.rst.snip

```diff
@@ -2,7 +2,7 @@
   GAPIC library for the {@metadata.fullName}
   ================================================================================
   
-  {@metadata.packageName} uses google-gax_ (Google API extensions) to provide an
+  gapic-{@metadata.packageName} uses google-gax_ (Google API extensions) to provide an
```

Comment:
I would much prefer you to have a string in the view model named something like `gapicPackageName` - this here amounts to the type of concatenation I would like to avoid.